File: sdk-kotlin/src/main/kotlin/org/sinou/pydia/sdk/client/model/FileNodeList.java
package org.sinou.pydio.sdk.client.model;

import com.pydio.cells.api.ui.FileNode;

import java.util.List;

public class FileNodeList {
    public int page;
    public int pageCount;
    public int offset;
    public int nodeCount;
    public List<FileNode> list;
}

File: sdk-kotlin/src/main/kotlin/org/sinou/pydia/sdk/client/model/FileNodeList.kt
package org.sinou.pydia.sdk.client.model

import org.sinou.pydia.sdk.api.ui.FileNode

class FileNodeList {
    var page = 0
    var pageCount = 0
    var offset = 0
    var nodeCount = 0
    var list: List<FileNode>? = null
}